pkgs/create-neon/data/templates/manifest/scripts.json.hbs

```diff
@@ -0,0 +1,12 @@
+{
+  "test": "{{#eq packageSpec.library.lang compare="ts"}}tsc && {{/eq}}cargo test",
+  "cargo-build": "{{#eq packageSpec.library.lang compare="ts"}}tsc && {{/eq}}cargo build --message-format=json > cargo.log",
+  "cross-build": "{{#eq packageSpec.library.lang compare="ts"}}tsc && {{/eq}}cross build --message-format=json > cross.log",
+  "postcargo-build": "neon dist < cargo.log",
+  "postcross-build": "neon dist -m /target < cross.log",
+  "debug": "npm run cargo-build --",
+  "build": "npm run cargo-build -- --release",
+  "cross": "npm run cross-build -- --release"{{#if packageSpec.library}},
+  "prepack": "neon update-platforms",
+  "version": "neon bump --binaries platforms && git add ."{{/if}}
```

Comment:
I think we have to do this. The default behavior of `npm version` is to automatically generate a commit and tag. Since our hook is creating more changes, we need to make sure to add all those changes before npm creates the commit, or the commit will be incomplete.

Reply:
See https://docs.npmjs.com/cli/v8/commands/npm-version#description starting at the third paragraph:

> If run in a git repo, it will also create a version commit and tag. This behavior is controlled by git-tag-version (see below), and can be disabled on the command line by running `npm --no-git-tag-version version`. It will fail if the working directory is not clean, unless the `-f` or `--force` flag is set.
> 
> If supplied with `-m` or `--message` config option, npm will use it as a commit message when creating a version commit. If the message config contains `%s` then that will be replaced with the resulting version number.

Reply:
We make use of this in CI/CD e.g. in `release.yml`:

```yaml
      - name: Tag Release
        if: ${{ !inputs.dryrun }}
        id: tag
        shell: bash
        run: |
          git config --global user.name $ACTIONS_USER
          git config --global user.email $ACTIONS_EMAIL
          npm version -m 'v%s' '${{ (inputs.version == 'custom' && inputs.custom) || inputs.version }}'
          git push --follow-tags
          echo tag=$(git describe --abbrev=0) >> "$GITHUB_OUTPUT"
```

Reply:
(It trolls me *to no end* that Crock insisted on disallowing comments in JSON and Node insisted on using pure JSON for configuration.)